src/makeCalculator.js

```diff
@@ -37,7 +37,49 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calcObject = {
+    operate(callback, number) {
+      return callback(number);
+    },
+
+    add(value) {
```

Comment:
only the reset and operation methods should return the current object
please check the example again
<img width="552" alt="Screenshot 2023-10-02 at 13 30 51" src="https://github.com/mate-academy/js_advanced_calculator/assets/64746943/2daa4537-6314-4247-8789-b6511fc963c9">